conf/extra-modules-example.yaml

```diff
@@ -148,6 +148,18 @@ modules:
 # notifies opers of user disconnections on remote servers.
 #- ServerNoticeRemoteQuit
 
+# StatsOnlineOpers: Provides the ONLINEOPERS STATS type, which lists all opers
```

Comment:
We should update the oper documentation as well then. Cause they're all in lowercase there.